examples/nlp/albert_squad_pytorch/const.yaml

```diff
@@ -0,0 +1,42 @@
+# After 2 epochs, model should hit 85.76/88.87 F1/EM
+description: ALBert_SQuAD_PyTorch_1gpu
+hyperparameters:
+    global_batch_size: 2
+    learning_rate: 5e-5
+    model_type: 'albert'
+    adam_epsilon: 1e-8
+    weight_decay: 0
+    num_warmup_steps: 13220  # 10% of total training
+    max_seq_length: 384
+    doc_stride: 128
+    max_query_length: 64
+    n_best_size: 20
+    max_answer_length: 30
+    null_score_diff_threshold: 0.0
+    max_grad_norm: 1.0
+    num_training_steps: 132198 # This is the number of optimizer steps. Train for 2 epochs
+    do_lower_case: true
+    use_radam: false
+resources:
+    slots_per_trial: 1
+searcher:
+    name: single
+    metric: f1
+    max_length:
+        records: 264396
+    smaller_is_better: false
+min_validation_period:
+    records: 80000
+data:
+    pretrained_model_name: "albert-xxlarge-v2"
+    use_bind_mount: True
+    bind_mount_path: /mnt/data
+    task: "SQuAD2.0"  # SQuaD 2.0 has 132198 example.
+entrypoint: model_def:AlbertSQuADPyTorch
+optimizations:
+    aggregation_frequency: 24
+max_restarts: 0
+bind_mounts:
```

Comment:
Just confirming here that there is nothing the user has to do with respect to bind mounts and this is already handled by just running the `det e create` command?

Reply:
Hmm, I guess for a normal user's setup, this won't work correctly because the bind mounted directory (`/home/ubuntu/dtrain-fsx/`) won't exist.

Extracting the features takes 30+ minutes, so we really want to cache it by default. I changed it so that by default we bind mount to `/tmp` because that should always exist